tests/Oryx.Integration.Tests/Node/NodeNuxtJsAppTest.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.Oryx.Integration.Tests
     [Trait("category", "node")]
     public class NodeNuxtJsAppTest : NodeEndToEndTestsBase
     {
-        public const string AppName = "hackernews-nuxtjs";
+        public const string AppName = "helloworld-nuxtjs";
```

Comment:
is it related to the bcmath package? Or does this PR address two issues?

Reply:
It actually address the php extension and fixes the node integration tests, so 2 issues.